examples/weather/fcn_afno/train_era5.py

```diff
@@ -130,23 +131,23 @@ def main(cfg: DictConfig) -> None:
     if dist.rank == 0:
         logger.file_logging()
         validation_datapipe = ERA5HDF5Datapipe(
-            data_dir="/data/test/",
-            stats_dir="/data/stats/",
-            channels=[i for i in range(20)],
-            num_steps=8,
-            num_samples_per_year=4,
-            batch_size=1,
+            data_dir=to_absolute_path(cfg.validation_dir),
+            stats_dir=to_absolute_path(cfg.stats_dir),
+            channels=list(range(cfg.num_channels)),
```

Comment:
If we are configuring this, is it possible to make this a list of channels instead of number or channels in the yaml?

Reply:
Good point - done.